src/Microsoft.Data.SqlClient/netcore/ref/Microsoft.Data.SqlClient.csproj

```diff
@@ -28,6 +28,5 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Azure.Identity" Version="$(AzureIdentityVersion)" />
-    <PackageReference Include="System.Text.Json" Version="$(SystemTextJsonVersion)" />
```

Comment:
This is required as System.Text.Json is exposed by SqlJson API

Reply:
@apoorvdeshmukh @deepaksa1 can you confirm this?

Reply:
Yes, this is correct.